packages/build/src/plugins_core/functions/zisi.js

```diff
@@ -14,8 +14,7 @@ export const getZisiParameters = ({
   repositoryRoot,
 }) => {
   const nodeVersion = childEnv.AWS_LAMBDA_JS_RUNTIME
-  const isManifestEnabled = isRunningLocally || featureFlags.buildbot_create_functions_manifest === true
-  const manifest = isManifestEnabled ? join(functionsDist, 'manifest.json') : undefined
+  const manifest = join(functionsDist, 'manifest.json')
```

Comment:
The following line should be removed too: https://github.com/netlify/build/blob/2ae0595e3c924211d3190cb4ad3bf24562d5f031/packages/build/src/core/feature_flags.js#L17